wolfcrypt/src/ecc.c

```diff
@@ -10275,6 +10506,27 @@ static int wc_ecc_import_raw_private(ecc_key* key, const char* qx,
                 WC_TYPE_UNSIGNED_BIN);
         }
     }
+#elif defined(WOLFSSL_XILINX_CRYPT_VERSAL)
+    if (err == MP_OKAY) {
+        const word32 key_size = key->dp->size;
+        word32 buf_size = sizeof(key->key_raw);
+        err = wc_export_int(key->pubkey.x, key->key_raw, &buf_size, key_size, WC_TYPE_UNSIGNED_BIN);
+        if (err == MP_OKAY) {
+            const word32 offset = buf_size;
+            buf_size = sizeof(key->key_raw) - offset;
```

Comment:
This section is a little hard to read, is there no possible way for offset to be larger than the sizeof(key->key_raw) and cause an underflow on the unsigned buf_size?. Please also use cammel case when naming variables, keySize, bufSize.